roles/rudder_agent/tasks/configure.yml

```diff
@@ -1,7 +1,34 @@
 - name: Define policy server
   ansible.builtin.copy:
     content: "{{ policy_server }}"
-    dest: "/var/rudder/cfengine-community/policy_server.dat"
+    dest: "{{ policy_server_dat_file }}"
     owner: root
     group: root
     mode: '0644'
+
+- name: Check if agent.conf file exist
+  file:
+    path: "{{ rudder_agent_config_file }}"
+    state: touch
+    owner: root
+    mode: 0644
+    modification_time: preserve
+    access_time: preserve
+
+- name: Set custom Rudder ports for 6 (min) version
+  replace:
+    path: "{{ policy_server_dat_file }}"
+    regexp: "^x=rudder.example.com:(.*)$"
```

Comment:
I don't get the `x=` here, and no the `rudder.example.com` either, shouldn't this match the currently configured policy server name?

Reply:
Indeed, I had simply based myself on what is referenced in the documentation.

Reply:
Doc: https://docs.rudder.io/reference/6.2/administration/port.html#_set_connection_port_on_the_agent